### PR TITLE
style: fixed some small details in codes

### PR DESCRIPTION
**Pull request by hk-mp5a3**

### CODE REVIEW

**Review by yifeili98 — APPROVED** [review]
Good job!

File: src/InsertData.py [hunk 1]
```diff
@@ -17,18 +17,18 @@
 from pymongo import errors as mongoerrors
 from pathlib import Path
 
-logging.basicConfig(filename = '../log/' + 
-                str(datetime.datetime.now()).replace(' ', '_').replace(':', '')[:17] + '.log', 
-                level=logging.INFO, 
-                format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+logging.basicConfig(filename='../log/' +
```
Comment (kis87988) [inline src/InsertData.py:20]: I am thinking about we should have logging module... what do you think?

Comment (yifeili98) [reply src/InsertData.py:20]: +1

Comment (hk-mp5a3) [reply src/InsertData.py:20]: yes, that would be cool. maybe we can do it in the next PR?

**Review by kis87988 — APPROVED** [review]

**Pull request merged**